### PR TITLE
Add pygraphblas

### PR DESCRIPTION
**Pull request by eriknw**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

This package does vendor some C code from a version of LAGraph, but it does not appear to include the full license (which is BSD-2).  See:
https://github.com/michelp/pygraphblas/tree/master/pygraphblas/cdef/LAGraph
https://github.com/GraphBLAS/LAGraph

For now, how should we add the LAGraph license?

@michelp, would you like to be added as a maintainer so you can see activity and maybe help answer `pygraphblas` questions if they arise?

CC @jim22k

### CODE REVIEW

File: recipes/pygraphblas/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  license_file: LICENSE
+  summary: 'GraphBLAS for Python'
+  description: |
+    pygraphblas is a Python wrapper around the GraphBLAS API.
+  dev_url: https://github.com/michelp/pygraphblas
+
+extra:
+  recipe-maintainers:
+    - eriknw
+    - jim22k
```
Comment (scopatz) [inline recipes/pygraphblas/meta.yaml:51]: CC @jim22k - are you willing to be a maintainer?

Comment (jim22k) [reply recipes/pygraphblas/meta.yaml:51]: Yes

**Comment by eriknw** [comment]
This is ready.  The license of the vendored code is now included in the 3.4.0 release.  Thanks for being responsive and accommodating @michelp

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**